test/testdata/resolver/nilable_untyped.rb

```diff
@@ -0,0 +1,39 @@
+# typed: false
+#
+# The type_member errors don't get re-raised in the fast-path
+# disable-fast-path: true
+
+
+Bad = T.type_alias {T.nilable(T.untyped)}
+                  # ^^^^^^^^^^^^^^^^^^^^ error: `T.nilable(T.untyped)` is the same as `T.untyped`
```

Comment:
These are minor suggestions, but i prefer writing the assertions like this so that my editor doesn't complain about misaligned indents. Non-blocking.

```suggestion
#                   ^^^^^^^^^^^^^^^^^^^^ error: `T.nilable(T.untyped)` is the same as `T.untyped`
```

Reply:
Sure, happy to change them to trailing assertions.